### PR TITLE
Add bitcoin estimates and registry_url

### PR DESCRIPTION
**Pull request by pcothenet**
### What

- Adding bitcoin endpoints
- Adding registry_url in endpoints

### SDK Release Checklist

- [x] Have you added an integration test for the changes?
- [x] Have you built the gem locally and made queries against it successfully?
- [x] Did you update the changelog?
- [x] Did you bump the package version?
- [x] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: lib/patch_ruby/models/order.rb [hunk 1]
```diff
@@ -221,7 +230,7 @@ def valid?
       return false if @mass_g < 0
       return false if @production.nil?
       return false if @state.nil?
-      state_validator = EnumAttributeValidator.new('String', ["draft", "placed", "complete", "cancelled"])
+      state_validator = EnumAttributeValidator.new('String', ["draft", "placed", "processing", "complete", "cancelled"])
```
Comment (pcothenet) [inline lib/patch_ruby/models/order.rb:233]: @bspellacy take-away is that we need to update those SDKs more often. Long-term I'm wondering if we could get to automate the submission of PRs upon updates to the Swagger file.

Comment (pcothenet) [reply lib/patch_ruby/models/order.rb:233]: cc @kleinjm

**Review by bspellacy — APPROVED** [review]

**Review by bspellacy — CHANGES_REQUESTED** [review]
Looks like we need to:
* Update the changelog
* Update the gem version

**Review by bspellacy — APPROVED** [review]
Looks good to me!

**Pull request merged**